compress/gzip/doc.go

```diff
@@ -5,6 +5,24 @@
     path: compress/gzip
     functions:
       decompress(data) bytes
-        Decompress the data, returning a bytes object containing the uncompressed data.
+        Return a bytes object containing the uncompressed data.
+				params:
+					data string, bytes
+						data can be a string or bytes of compressed gzip data
+				examples:
+					basic
+						decompress gzip contents
+						code:
+							# load("compress/gzip.star", "gzip")
+							# compressed = "some compressed data"
+							# data = gzip.decompress(compressed)
+					download:
+						download a gzip file & decompress the contents
+						code:
+							load("compress/gzip.star", "gzip")
+							load("http.star", "http")
+							url = "http://www.sbeams.org/sample_data/Proteomics/yeast_orfs.new.20040422.gz"
+							raw = http.get(url).body()
+							data = gzip.decompress(raw)
```

Comment:
So this example works, but it is downloading a file from a biology research group.

I'm not sure that we want to actually put this URL in here, but in order to have the example tests pass, this would need to stay in.

The other option is to just comment this out the way that the above "basic" example is commented out.

Reply:
My worry is that someone will look at the fully commented out example and be confused about... why everything is commented out.

We may need some notation to tell starlib "don't run this example"